torch/_inductor/codecache.py

```diff
@@ -326,16 +327,15 @@ def get_lock_dir() -> str:
     return lock_dir
 
 
+def sha256_hash(data: bytes) -> str:
+    return base64.b32encode(hashlib.sha256(data).digest())[:51].decode("utf-8").lower()
+
+
 def code_hash(code: Union[str, bytes], extra: str = ""):
     hashing_str = code if isinstance(code, bytes) else code.encode("utf-8")
     if extra != "":
         hashing_str = hashing_str + b"||" + extra.encode("utf-8")
-    return (
-        "c"
-        + base64.b32encode(hashlib.sha256(hashing_str).digest())[:51]
-        .decode("utf-8")
-        .lower()
-    )
+    return "c" + sha256_hash(hashing_str)
```

Comment:
The prefix character seems to be the existing scheme here to differentiate different kinds of hashes for different types of cached objects. Probably I should use an enum. TODO.

Reply:
I was sticking with the convention already started, but can definitely turn it into a a more descriptive string. There's also an existing precedent for adding an extension, e.g., "cg" which I guessed means "code graph"?